webapp/pom.xml

```diff
@@ -94,7 +94,7 @@
          <plugin>
             <groupId>org.mortbay.jetty</groupId>
             <artifactId>maven-jetty-plugin</artifactId>
-            <version>6.1.10</version>
+            <version>6.1.26</version>
```

Comment:
I opened TRUNK-4638 for upgrading to Jetty 8. (I'm not claiming the ticket
though, so anybody that wants to take this feel free)

On Wed, Mar 11, 2015 at 5:22 AM, Ryan Crichton <notifications@github.com>
wrote:

> In webapp/pom.xml
> <https://github.com/openmrs/openmrs-core/pull/1393#discussion_r26197683>:
>
> > @@ -94,7 +94,7 @@
> >           <plugin>
> >              <groupId>org.mortbay.jetty</groupId>
> >              <artifactId>maven-jetty-plugin</artifactId>
> > -            <version>6.1.10</version>
> > +            <version>6.1.26</version>
>
> Yeah, I didn't want to be too disruptive but the latest version of jetty
> worked for me to. I agree with @djazayeri <https://github.com/djazayeri>
> another issue would be the best place to do this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openmrs/openmrs-core/pull/1393/files#r26197683>.
>